staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -374,6 +374,13 @@ func LogLibvirtLogLine(logger *FilteredLogger, line string) {
 	pos := strings.TrimSpace(fragments[3])
 	msg := strings.TrimSpace(fragments[4])
 
+	//TODO: implement proper behavior for unsupported GA commands
+	// by either considering the GA version as unsupported (don't send any command)
+	// or just don't send commands which not supported
+	if strings.Contains(msg, "unable to execute QEMU agent command") {
```

Comment:
How about making it a warning? I would also consider only logging them in debug mode. You can do that by checking if the verbosityLevel is >=4 and just return here if it is below 4.

Reply:
@rmohr Done